deploy/template.yaml

```diff
@@ -77,6 +77,8 @@ parameters:
   value: 15Gi
 - name: REPLICAS_COUNT
   value: "3"
+- name: IMAGE_SERVICE_BASE_URL
+  value: https://api.integration.openshift.com/api/assisted-images
```

Comment:
I think this should probably be empty and required like we do in [assisted service](https://github.com/openshift/assisted-service/blob/master/openshift/template.yaml#L8-L10)

That would mean adding the parameter to the saas file first though.